www/src/routes/Landing.tsx

```diff
@@ -55,11 +57,13 @@ const Intro: FC = () => {
                                 CompE+ is a student-led pilot program to provide support for computer engineering students looking for internships
                             </Typography>
                         </Grid>
-                        <Grid item>
-                            <Button variant='contained' color='primary' onClick={() => console.log('clicked Get Started button')}>
-                                {'Get Started'}
-                            </Button>
-                        </Grid>
+                        {!isAuthenticated && (
+                            <Grid item>
+                                <Button variant='contained' color='primary' onClick={() => console.log('clicked Get Started button')}>
+                                    {'Get Started'}
```

Comment:
What is the point of the `Get Started` button? Why do we need to hide it when the user is logged in?

Reply:
I think its supposed to ask someone to log-in

Reply:
^ yup, the `Get Started` button is just to bring them to the Login page